test/cypress/integration/request-to-go-live/organisation_name_spec.js

```diff
@@ -1,40 +1,22 @@
+'use strict'
+
 const lodash = require('lodash')
+const utils = require('../../utils/request_to_go_live_utils')
+const variables = utils.variables
```

Comment:
You could use object destructuring to save having to initialise the id variables later.

So:
```const { userExternalId, gatewayAccountId, serviceExternalId } = utils.variables```

Reply:
Thanks. I was gonna do this but opted to follow the other integration tests for consistency. I guess we could start refactoring once we're done, so we're not all working on the same common modules at once.